doc/Ontology.md

```diff
@@ -0,0 +1,208 @@
+as possible to help make sure your entities are uniquely defined if they
+are unique from other entities that share the same name. Be careful to define your
+attributes accurately so merges across knowledge graphs will map equivalent resources
+to each other.  This means all attributes must match for two nodes to be considered the
+same resource.  Due to this matching requirement, the attributes for each currently 
+defined entity are detailed in the following sections.
+
+### Attributes of the Users Entity
+
+Attribute | Description | Sample | Challenges
+---       | ---         | ---    | ---
+entity | Users |  | 
+name | The name should simply be User | `User` | Considerations for different types of users is not being made. Groups are also not being considered.
+
+
+### Attributes of the API Entity
+
+Attribute | Description | Sample | Challenges
+---       | ---         | ---    | ---
+entity | The entity is defined above. | API | 
+type | The platform type of the resource or controller. | Kubernetes, REST, Ansible, vSphere | For most ACM resources, the type should be `Kubernetes`.
+group | For the Kubernetes type, this is the kube group value.  For other types it is TBD and will be defined by those domain experts | policy.open-cluster-management.io, cluster.open-cluster-management.io |  
+kind | The kubernetes kind when the type is Kubernetes. When using other types it will be defined by those domain experts. | Policy, ManagedCluster, TBD: Playbook | 
+name | The name that identifies this entity | Pod name, policy name | If the name can be variable use the syntax `{cluster-name}` or `insights-client-{unique-id}` or `{user-defined}`.
+namespace | The k8s namespace of the resource | default |
+onHub | Boolean that is true if this entity exists on the ACM hub | true/false | If the namespace can be variable use the syntax `{cluster-name}`.
+onManaged | Boolean that is true if this entity is on managed clusters | true/false |
+
+### Attributes of the Intermediate Resource Entity
+
+Attribute | Description | Sample | Challenges
+---       | ---         | ---    | ---
+entity | The Intermediate Resource Entity. | Intermediate Resource | 
+type | The platform type of the resource or controller. | Kubernetes, REST, Ansible, vSphere | For most ACM resources, the type should be `Kubernetes`.
+group | For the Kubernetes type, this is the kube group value.  For other types it is TBD and will be defined by those domain experts | policy.open-cluster-management.io, cluster.open-cluster-management.io |  
+kind | The kubernetes kind when the type is Kubernetes. When using other types it will be defined by those domain experts. | ConfigurationPolicy, Pod, Secret, ConfigMap | 
+name | The name that identifies this entity | Pod name, ConfigurationPolicy name | If the name can be variable use the syntax `{cluster-name}` or `insights-client-{unique-id}`.
+namespace | The k8s namespace of the resource | default | Leave empty for cluster scoped resources.
```

Comment:
`Leave empty` -> does it mean do not use namespace attribute at all or keep it '' ?

Reply:
I'm going to pick do not specify the namespace attribute.  That's probably a change from what I originally was thinking.